lib/alchemy/elements_finder.rb

```diff
@@ -54,8 +54,7 @@ def elements(page_version:)
     def find_elements(page_version)
       return Alchemy::ElementsRepository.none unless page_version
 
-      elements = Alchemy::ElementsRepository.new(page_version.elements.select(&:public?))
-      elements = elements.not_nested
+      elements = page_version.element_repository.visible.not_nested
```

Comment:
Ah. This makes sense. Can we move this into its own PR? This is not related to this topic. And I want the changelog to mention this change.

Reply:
#2313